app/services/sign_in/constants/service_account_access_token.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module SignIn
+  module Constants
+    module ServiceAccountAccessToken
```

Comment:
We might as well go ahead and copy the rest of the access token constants into this and refer to this set of constants wherever we use the service account functionality

If you do a global search for `Constants::AccessToken` you should be able to see everywhere that needs to be changed to the service account access token without too much difficulty (I only see ~50 usages so not too huge)